src/main/python/rlbot/setup_manager.py

```diff
@@ -14,6 +14,9 @@
 
 from rlbot.utils.structures import game_data_struct
 
+from rlbot.utils.queue import Queue as _queue
+mp.Queue = _queue
```

Comment:
I'd recommend against modifying module objects and instead changing uses of `mp.Queue` to `Queue`

Reply:
Ok. Sorry, I won't be able to update this until monday